lib/capsize/application.rb

```diff
@@ -0,0 +1,31 @@
+module Capsize
+  module Application
+    def self.load_tasks
+      Application.load_tasks_in_isolation do
+        require 'capistrano/all'
+        cap = Capistrano::Application.new
+        Rake.application = cap
+        cap.init
+        cap.load_rakefile
+        cap.tasks
+      end
+    end
+
+    def self.load_tasks_in_isolation
+      read, write = IO.pipe
```

Comment:
You should probably use the block form of `IO.pipe` so that you don't have to manage closing the pipe yourself.